src/pages/workspaces/workspace/analysis/AppLauncher.js

```diff
@@ -227,6 +228,8 @@ const ApplicationLauncher = _.flow(
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [googleProject, workspaceName, runtimes, bucketName])
 
+  if (!busy && runtimeStatus === undefined && runtime === undefined) Nav.goToPath(analysisTabName, { namespace, name })
```

Comment:
I dont think we need to check runtime status here, if runtime is undefined its status will trivially be so, correct?

Reply:
You're right!